  file does not contain alignment data
  [Exception type: ValueError, raised in libcalignmentfile.pyx:947]
